tools/topology/topology2/cavs/cavs-sdw.conf

```diff
@@ -134,6 +152,37 @@ Object.Pipeline {
 		}
 	}
 
+	host-copier-gain-mixin-playback."3" {
+		index 3
+
+	Object.Widget.copier.1 {
+		stream_name "volume playback 1"
```

Comment:
naming seems weird here, what does copier have to do with volume?

Reply:
> naming seems weird here, what does copier have to do with volume?

Renamed to `"sdw amplifiers"`